css/style.css

```diff
@@ -16,4 +16,197 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+/* nav Style */
+nav {
+    /* border: 2px solid red ; */
+    position: fixed;
+    width: 100%;
+    z-index: 1;
+    background-color: whitesmoke;
+    
+}
+
+#logo img{
+    width: 37px;
+    height: 37px;
+}
+
+ul li {
+    list-style-type: none;
+}
+
+li a {
+    text-decoration: none;
+    font-size: 1rem;
+    font-weight: 500;
+    color: lightslategray;
+    /* border: 3px solid lime; */
+    margin: 25px 35px 5px 0px;
+    
+}
+
+a:hover {
+    color: darkslategray
+    
+}
+
+.navContainer {
+    display: flex;
+    flex-flow: wrap;
+    justify-content: center;
+    align-items: center;
+}
+
+#logo {
+    /* margin-left: 10rem; */
+    /* margin-right: 38rem; */
+    margin-right: 25vw;
+}
+
+#linkOne {
+    margin-left: 14rem;
+    color:black;
+    
+}
+#linkOne,
+#linkTwo, #linkThree, #linkFour, #linkFive, #linkSix {
+    margin: 0px 15px;
+}
```

Comment:
instead of targeting the elements by id, as all these elements are meant to the same anyway, consider creating a class to capture this styling instead.